tests/utils/waitForAll.test.tsx

```diff
@@ -311,7 +311,7 @@ it('large atom count', async () => {
     atom((getter) => {
       const data = createArray(count)
       const atoms = data.map(chunksFamily)
-      const values = waitForAll(atoms)
+      const values = waitForAll(atoms as any /* FIXME better typing? */)
```

Comment:
I don't think numbers go well (`i` & `count`) with Promises in `waitForAll`!
So I think it's better to delete the FIXME comment!

Reply:
Or changing the test behaviour to async, which broke the test for me (basically)!

Reply:
Thanks for the hint.
72333c3 looks good?

Reply:
Now I think it's great!

Reply:
That was my bad: fixed in b587a6d and c9bf5a3